apps/mobile/src/features/earn/sbtc-card.tsx

```diff
@@ -13,8 +13,8 @@ export function SbtcCard() {
   return (
     <EarnCard
       title={t`Earn with SBTC`}
-      minYield={t`6`}
-      maxYield={t`8%`}
+      minYield="6"
```

Comment:
Nice cleanup here 👍 we wanted to update these via CrowdIn at one point but not needed now